### DISCUSSION TITLE
Transition Group Gravity

### DISCUSSION THREAD

**Discussion by eli-crow** [vuejs/rfcs#261] [RFC Discussions]
Add a `gravity` prop to `<transition-group>` to prevent unintuitive "jumping" before `-move` transitions in lists whose items stack to the right or bottom. The prop would change the point on the ClientRect used to measure how far the item has moved. This is most salient when the child element changes in size.

It's a small feature, but I'm happy to contribute in some small way, even hypothetically! Been loving Vue as a UX designer for years now. Please help me consider any missing angles and let me know how I can improve the RFC. Cheers!

- [Pull Request](https://github.com/vuejs/rfcs/pull/262)
- [Rendered RFC](https://github.com/eli-crow/rfcs/blob/transition-group-gravity/active-rfcs/0000-transition-group-gravity.md)
- [Codesandbox with demo implementation](https://codesandbox.io/s/transition-group-bottom-anchored-example-nv3bh?file=/src/App.vue)

**Comment by LinusBorg** [comment]
Tip: give a short overview here so people get a basic idea and can be motivated to read the RFC.
